### ISSUE TITLE
Articles and Pages not accessible

### ISSUE THREAD

**Issue by denniserdmann** [terminal42/contao-changelanguage#216]
Contao: 4.9.37
PHP: 8.1.13
Changelanguage: 3.3.7

After updating contao via manager last month, our client complains that she can not access articles and pages via backend.
As the error logs suggest, this has something to do with the changelanguage extension:

```
[2023-01-09 15:38:27] request.INFO: Matched route "contao_backend". {"route":"contao_backend","route_parameters":{"_route":"contao_backend","_scope":"backend","_token_check":true,"_controller":"Contao\\CoreBundle\\Controller\\BackendController::mainAction"},"request_uri":"https://www.client.com/contao?do=article&ref=6KOnZftj","method":"GET"} []
[2023-01-09 15:38:27] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "getSession" of class "Symfony\Component\HttpFoundation\RequestStack"." at /vendor/terminal42/contao-changelanguage/src/EventListener/BackendView/PageViewListener.php line 28 {"exception":"[object] (Symfony\\Component\\ErrorHandler\\Error\\UndefinedMethodError(code: 0): Attempted to call an undefined method named \"getSession\" of class \"Symfony\\Component\\HttpFoundation\\RequestStack\". at /vendor/terminal42/contao-changelanguage/src/EventListener/BackendView/PageViewListener.php:28)"} []
```

Do you have an idea whats the problem here?

**Comment by fritzmg** [comment]
This is a regression caused by https://github.com/terminal42/contao-changelanguage/commit/2cc30991b6118a6c874e16c598feeedbadad889b

Symfony 4.4 has no `RequestStack::getSession` method. This needs to be changed to

```php
System::getContainer()->get('request_stack')->getCurrentRequest()->getSession()
```

**Comment by denniserdmann** [comment]
@fritzmg thanks you!

**Comment by aschempp** [comment]
Fixed in 3.3.8
